clamav/test_clamav_format.sh

```diff
@@ -11,7 +11,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-set -euo pipefail
+set -eu
```

Comment:
we can keep `-o pipefail` there, it doesn't hurt. It helps when somebody adds `|` into script in future

Reply:
Ok. Updated.